src/assets/javascript/main.js

```diff
@@ -0,0 +1,60 @@
+import "../../assets/main.css"; 
+const Button = require("../../components/button");
+document.querySelector("#Buttons").insertAdjacentHTML("beforeend", Button());
+
+const inputReuslt = document.querySelector(".js-result");
+const patternElements = document.querySelectorAll(".js-btn-pattern");
+const ClearAllElement = document.querySelector(".js-btn-ClearAll");
+const lessMoreElement = document.querySelector(".js-btn-less-more");
+const delElement = document.querySelector(".js-btn-Del");
+const equaElement = document.querySelector(".js-btn-equal");
+
+//-------------------- textContent of all buttons-----------------------------//
+ClearAllElement.textContent = "C";
+lessMoreElement.textContent = "+/-";
+delElement.textContent = "«";
+equaElement.textContent = "=";
+const actions_patterButtons = [
```

Comment:
i added the **buttons' textContent** using this array `const actions_patterButtons = [...]`, because i only saw this solution(for now) and 'cause the function `Button(textContent)` it's returning the whole html.
what do you think doing it this way ?